python/src/aiconfig/editor/client/src/components/notifications/NotificationContext.tsx

```diff
@@ -0,0 +1,13 @@
+import { createContext } from "react";
+import { AIConfigEditorNotification } from "./NotificationProvider";
+
+/**
+ * Context for providing showNotification method throughout the editor
+ */
+const NotificationContext = createContext<{
```

Comment:
Nit: can we put these files in the `/context` folder since we're not really making 'components': https://github.com/lastmile-ai/aiconfig/tree/10da93839d85f4dee2ccd5428c9bc10104eb2246/python/src/aiconfig/editor/client/src/contexts

Reply:
Hmm, I have it colocated here under /notifications folder so it's right beside the provider. I think that makes more sense than having it in a separate folder a few levels away. Also if it's not here, this folder just has one file, which is kinda lonely :(